### ISSUE TITLE
Google Photos unlimited storage - google_experience.xml

### ISSUE THREAD

**Issue by Tachi107** [GrapheneOS/os-issue-tracker#1049]
When using Google Photos on GrapheneOS on my Pixel 4 I'm unable to get unlimited storage, while it should be available for Pixel 5 and older devices.

I've looked a bit into the issue, and I believe that it is caused by the absence of some features in `product/etc/sysconfig/google_experience.xml`, like the ones in `product/etc/sysconfig/pixel_<year>_exclusive.xml` (in my case, GrapheneOS is missing `com.google.android.apps.photos.PIXEL_2019_PRELOAD`).

I'm not sure that adding that feature to the xml file would fix the issue as I didn't test it.

For a bit of context you can take a look at the brief discussion I had with @flawedworld on Matrix [here](https://matrix.to/#/!SayHlEYXdrpSerhLMC:matrix.org/$baMX9I-IgKlKrQlrvia-vFNNQAZSnM6EN-UIcuWovtU?via=matrix.org&via=tchncs.de&via=kde.org)

**Comment by Zanthed** [comment]
They cancelled unlimited storage in favour of Google One, no?

https://blog.google/products/photos/storage-changes/

**Comment by Tachi107** [comment]
Yes, but as they promoted free unlimited storage as selling points for the Pixels they had to keep it free for gen 1 to 5 - https://support.google.com/photos/answer/6220791?co=GENIE.Platform%3DAndroid&oco=1#storage

**Comment by Tachi107** [comment]
To clarify, it seems that once uploaded to Drive pictures don't take up storage space, but the Photos app itself doesn't mention it anywhere and it is quite confusing

**Comment by lbdroid** [comment]
Isn't google photos an anti-feature? It doesn't do anyone any good to store their photos with google, but it does google a lot of good by giving them access to much more of your data. Its really easy and inexpensive to set yourself up a Nextcloud server and sync your phone storage to it. Even can be done on your home router if you have a decently powerful one so it won't even draw extra power.

**Comment by Zanthed** [comment]
@lbdroid Not useful or relevant to the issue. Please don't post unproductive and unnecessary comments on our issue trackers.

**Comment by lbdroid** [comment]
How is security not a relevant issue to a security hardened OS?

**Comment by thestinger** [comment]
Fixing things like this is something we want to do. Giving people the option to use Google apps and services is not in any way opposed to the goals of GrapheneOS. It's not an anti-Google project and that's missing the point of the privacy/security focus. We would recommend that people use E2EE services for storage, sharing, messaging, etc. but that doesn't me we want non-E2EE ones to be non-functional. It's not appropriate to argue about that here. The issue tracker is for improving the project not discussions about anything else.

**Comment by flawedworld** [comment]
This will be dealt with as part of the device support overhaul.

**Comment by heyakyra** [comment]
_**EDIT:** Google support forums product expert has [escalated the issue with the Google Photos engineering team](https://support.google.com/photos/thread/425723871?hl=en&msgid=430674944) and also confirm that simply **renaming the file extensions to `.CR2`** is a more elegant workaround!_

It seems that the latest version which supports the OG Pixel is [QP1A.191005.007.A1.2019.10.07.21](https://github.com/GrapheneOS/platform_manifest/releases/tag/QP1A.191005.007.A1.2019.10.07.21) ([changelog](https://grapheneos.org/releases#2019.10.07.21)), so this patch would not have made it in. 

From this comment:

> To clarify, it seems that once uploaded to Drive pictures don't take up storage space, but the Photos app itself doesn't mention it anywhere and it is quite confusing

Does that mean that images backed up in Google photos still won't take up any storage capacity, just the UI won't indicate that ahead of time?

Asking because [support for CR3 image files landed in Android 11](https://cs.android.com/android/_/android/platform/external/piex/+/a1b85d566521a4a03c7c715d2851c4272da4e301), so I want to see if this will allow me to skip doing batch conversions using the lovely [dnglab](https://github.com/dnglab/dnglab/issues/447) (great program, but would rather save originals).
